### PR TITLE
Cancel property type edit confirmation modal

### PR DESCRIPTION
**Pull request by luisbettencourt**
## 🌟 What is the purpose of this PR?

This PR makes it so the confirmation modal for deleting property types changes when editing property types.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203363157432094/1203607109530575/f) _(internal)_

## 📹 Demo

demo-edit

image

### CODE REVIEW

File: packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card/shared/property-type-form/expected-value-selector/custom-expected-value-builder/array-expected-value-builder.tsx [hunk 1]
```diff
@@ -307,6 +307,7 @@ export const ArrayExpectedValueBuilder: FunctionComponent<
         <DeleteExpectedValueModal
           expectedValueType="array"
           popupState={deleteModalPopupState}
+          editing={editingExpectedValueIndex !== undefined && !index.length}
```
[the remark this answers is not in the document]
Comment (luisbettencourt) [reply packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card/shared/property-type-form/expected-value-selector/custom-expected-value-builder/array-expected-value-builder.tsx]: Exactly, added a comment here 561bcab750d0f7b98718a10b08c6f96b26b06580

**Review by nathggns — APPROVED** [review]

**Comment by luisbettencourt** [comment]
@nonparibus we may also want to list the changes that are going to be reverted at some point.

**Comment by nonparibus** [comment]
@luisbettencourt Can we just remove the "and will need to be individually re-added should you wish to restore them"? I think it will read more simply than the original proposed copy. Thank you!

**Review by nathggns — APPROVED** [review]

**Comment by nathggns** [comment]
Approved assuming @nonparibus is cool with the text only being removed when editing

**Pull request merged**